README.md

```diff
@@ -17,6 +15,52 @@ This will launch your browser, where the demo page will be displayed (at http://
 
```

Comment:
Would it make sense to have the demo page open on start? You could use https://www.npmjs.com/package/opener to make sure it works cross platform.